src/Util.cc

```diff
@@ -55,8 +57,19 @@
   static const auto &ignstrtok = strtok_r;
 #endif
 
-static std::unique_ptr<ignition::common::SystemPaths> gSystemPaths(
-    new ignition::common::SystemPaths);
+/////////////////////////////////////////////////
+/// \brief Global instance of system paths for helper functions below
+///
+/// This uses the NeverDestroyed pattern to prevent static initialization and
+/// destruction order fiasco issues.
+ignition::common::SystemPaths* GetSystemPaths()
+{
+  static const
+    ignition::utils::NeverDestroyed<ignition::common::SystemPaths*> paths{
+      new ignition::common::SystemPaths()
+    };
+  return paths.Access();
```

Comment:
I was under the impression you can just do 
```c++
  static const ignition::utils::NeverDestroyed<ignition::common::SystemPaths> paths;
  return &paths.Access();
```

without the `new`.

Reply:
Ah, good point, I think this may have been a workaround for something `const`.  Will fix.